.github/renovate.json

```
@@ -1,20 +1,27 @@
 {
-  "schedule": ["every weekend"],
-  "labels": ["dependencies"],
+  "extends": [ "config:base", ":dependencyDashboardApproval" ],
+  "constraints": {
+    "node": "< 16"
```

Does this allow Node 15? Would like to exclude non-LTS (odd numbered) versions of NodeJS, just checking.

```suggestion
    "node": "< 15"
```

I think this is right for "14 or lower" (for Electron 12 compatiblity-ish).

---

Ahh I may have gotten that wrong then, but this should accept anything. As it's all handled by semver as far as I know